src/components/TeamMember.js

```diff
@@ -0,0 +1,33 @@
+import {
+  Box,
+  Card,
+  CardContent,
+  Typography,
+} from "@material-ui/core/";
+import useMyStyles from "../styles/MyStyles";
+import AccountCircleSharpIcon from "@material-ui/icons/AccountCircleSharp";
+
+const TeamMember = (props) => {
```

Comment:
destructurize props -> `({ id, url, firstName })` etc - you won't need repeat `props` in whole component

Reply:
Destructured props as suggested.